examples/mixture_models/dependent_density_regression.ipynb

```diff
@@ -2,16 +2,7 @@
  "cells": [
```

Comment:
<p>so much more readable with xarray magic...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>:) </p><p> </p><p>Updating the examples is being an amazing resource for me to both see the potential of xarray as well as where is more help and work needed. I don't need convincing on xarray usage, but there are still things where it surprises me. </p><p> </p><p>As a side not, I think somehting like <code>max_mixture_weights["K"].values</code>  will be equivalent to<code>max_mixture_weights.coords.to_index()</code> . The distinction between coord and variable is still something I don't grok yet, especially because of non-indexing coords, but it is cool to be able to treat coords as variables in many cases </p>